front/lib/models/assistant/actions/data_sources.ts

```diff
@@ -18,6 +18,10 @@ export class AgentDataSourceConfiguration extends WorkspaceAwareModel<AgentDataS
   declare parentsIn: string[] | null;
   declare parentsNotIn: string[] | null;
 
+  declare tagsQuery: "fixed" | "auto" | null;
```

Comment:
On `AgentRetrievalConfiguration` we use  "custom" should we use the same?

Reply:
Maybe `tagsFilter` more than `tagsQuery`? Or even `tagsMode` if we want to align with `topKMode` ?

Reply:
Agreed, will rename `declare tagsMode: "custom" | "auto" | null;`

=> picking "mode" over "filter because on my next PR that holds all the filtering logic from agent config to core I'm using tagsFilter already and I'd rather avoid confusion (https://github.com/dust-tt/dust/pull/10421/files)

Reply:
👍👍